treescript/src/treescript/github/client.py

```diff
@@ -0,0 +1,137 @@
+"""Treescript git functions."""
+import base64
+import logging
+from collections import defaultdict
+import os
+from string import Template
+from textwrap import dedent
+from typing import Dict, List, Optional, Union
+
+from simple_github import AppClient
+
+log = logging.getLogger(__name__)
+
+GITHUB_APP_ID = 390757  # https://github.com/apps/releng-treescript
+
+
+class GithubClient:
+    def __init__(self, owner, repo):
+        privkey = base64.b64decode(os.environ["SSH_KEY"]).decode("utf-8")
```

Comment:
> Presumably this is not actually a ssh key?

Yeah, I guess I was being lazy and trying to avoid updating cloudops-infra.. but let's do this properly.

> Also, I'm surprised that variable is available to the script

Good catch, I was testing by running the `treescript` binary directly, which I guess skips that code. I'll get this added to the config instead.

Once the apps RRA is done and we have the app ready and secret in SOPS, I'll trigger a proper staging release to test the overall system.